src/js/modules/infragistics.ui.editors.js

```diff
@@ -8132,7 +8184,23 @@
 			this._spin(this.options.spinDelta, true);
 		},
 		_handleSpinDownEvent: function () { // DateEditor
-			this._spin(-this.options.spinDelta, true);
+			this._spin(this._setNegativeDelta(this.options.spinDelta), true);
+		},
+		_setNegativeDelta: function (delta) {

```

Comment:
This means that down the chain we should keep note if we are spinning down or up, which is something I want to avoid. I don't think this will create some performance issues, knowing the fact that the spinDelta object contains 7 properties, so I suggest leave it that way.